ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,34 @@
+# Thanks for your help improving the project! #
+
+## Getting Help ##
+
+Github issues are for bug reports and feature requests. For questions about
+linkerd, how to use it, or debugging assistance, start by
+[asking a question in the forums](https://discourse.linkerd.io/) or join us on
+[Slack](https://slack.linkerd.io/).
+
+Full details at [CONTRIBUTING.md](CONTRIBUTING.md).
+
+## Filing a linkerd issue ##
+
+Issue Type:
+
+- [ ] Bug report
+- [ ] Feature request
+
+**What happened**:
+
+**What you expected to happen**:
+
+**How to reproduce it (as minimally and precisely as possible)**:
+
+**Anything else we need to know?**:
+
+**Environment**:
+- linkerd/namerd version, config files:
+- Platform, version, and config files (Kubernetes, DC/OS, etc):
+- Cloud provider or hardware configuration:
+- OS (e.g. from /etc/os-release):
```

Comment:
I think these last 4 bullets are rarely relevant.  I think we can ask for these if we need them.